CorsixTH/Src/th_gfx_sdl.cpp

```diff
@@ -740,12 +827,21 @@ bool render_target::take_screenshot(const char* sFile) {
                              pRgbSurface->pixels, pRgbSurface->pitch);
     SDL_UnlockSurface(pRgbSurface);
 
-    if (readStatus != -1) SDL_SaveBMP(pRgbSurface, sFile);
+    if (readStatus != -1) {
+      try {
+        write_rgb_png(width, height,
+                      static_cast<png_bytep>(pRgbSurface->pixels),
+                      pRgbSurface->pitch, file_path);
+        ok = true;
+      } catch (int zero) {
```

Comment:
This is c++, use an unnamed parameter instead of a bogus assignment.

Reply:
Done.

Don't think I ever used that before. Even with base classes I like to have a variable name for documentation purposes.